modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-ocp-oauth.adoc
+:_module-type: PROCEDURE
+[id="configure-ocp-oauth-identity-provider_{context}"]
+= Configuring {ocp} Oauth server as an identity provider in {product-title}
+
+[role="_abstract"]
+Use the instructions in this section to integrate the built-in {ocp} OAuth server as an identity provider for {product-title}.
+
+.Prerequisites
+* You must have permissions to configure identity providers in {product-title}.
+* You must have an Okta app configured for {product-title}.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Access Control*.
+. Open the *Add an Auth Provider* menu and select *{ocp} OAuth*.
```

Comment:
Don't use the ocp variable here. Its directly encoded as "OpenShift Auth"

![image](https://user-images.githubusercontent.com/5336227/143324124-4a385a93-37a9-449e-b1c2-91563970eb1d.png)